sys/include/net/icmp.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2015 José Ignacio Alamos <jialamos@uc.cl>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_icmp  ICMPV4
+ * @ingroup     net_ipv4
+ * @brief       Provides types related to ICMPv4
+ * @see         <a href="https://tools.ietf.org/html/rfc792">
+ *                  RFC 792
+ *              </a>
+ * @{
+ *
+ * @file
+ * @brief   ICMPv4 type and function definitions
+ *
+ * @author  José Ignacio Alamos <jialamos@uc.cl>
+ */
+#ifndef ICMPV4_H
+#define ICMPV4_H
+
+#include "byteorder.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+/**
+ * @brief   Echo request and response message format.
+ *
+ * @see <a href="https://tools.ietf.org/html/rfc792">
+ *          RFC 792, page 13
```

Comment:
It's page 14 and please provide an anchor link (https://tools.ietf.org/html/rfc792#page-14) to that page.